src/org/labkey/test/util/StudyHelper.java

```diff
@@ -389,6 +381,30 @@ public void publishStudy(String studyName, int expectedPipelineJobs, String subj
         _test.waitForPipelineJobsToComplete(expectedPipelineJobs, "Publish Study", false);
     }
 
+    public void advanceThroughPublishStudyWizard(List<StudyHelper.Panel> wizardPanels)
+    {
+        advanceThroughPublishStudyWizard(wizardPanels, false);
+    }
```

Comment:
Add an overload that takes a single panel. Seems like a common use case.

```suggestion
    public void advanceThroughPublishStudyWizard(StudyHelper.Panel wizardPanel, boolean selectAll)
    {
        advanceThroughPublishStudyWizard(List.of(wizardPanels), selectAll);
    }

    public void advanceThroughPublishStudyWizard(List<StudyHelper.Panel> wizardPanels)
    {
        advanceThroughPublishStudyWizard(wizardPanels, false);
    }
```

Reply:
Good call. Done.